clvm/serialize.py

```diff
@@ -26,6 +27,8 @@
 BACK_REFERENCE = 0xFE
 CONS_BOX_MARKER = 0xFF
 
+MAX_SAFE_BYTES = 2_000_000
```

Comment:
I don't see this constant being used anywhere. I think it would be reasonable to make it the default value of `max_size`

Reply:
Consider renaming it `DEFAULT_MAX_SAFE_BYTES`.